ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -323,64 +266,53 @@ const PanelMain = fp.flow(
 
     const currentRunningCost = machineRunningCost(analysisConfig);
 
-    const diskValidator = {
```

Comment:
Moved these inside getErrorMessageContent() because they are only used there: diskValidator (inlined), dataprocValidators (inlined), diskErrors, dataprocErrors